client/client.go

```diff
@@ -67,24 +68,30 @@ type ChargePayload struct {
 
 // Offer is the api message of an Offer.
 type Offer struct {
-	ID string
+	ID int
 
-	OfferType OfferType
-	// If OfferType is not Instance, InstanceID would be empty, otherwise InstanceID may be the sku of that instance.
 	// e.g. AWS: 9QH3PUGXCYKNCYPB, GCP: 0009-6F35-3126
-	InstanceID string
+	InstanceSKU string
+	// Allowed OfferType are Instance, RAM, CPU
+	OfferType OfferType
+	// If the offer type is Instance, the payload would be the information of that instance, otherwise this field will be nil
+	InstancePayload *OfferPayloadInstance
 
+	// Possible ChargeType are reserved, onDemand
 	ChargeType ChargeType
 	// Payload is present when the ChargeType is Reserved, otherwise nil.
 	ChargePayload *ChargePayload
 
+	// RegionList is the region that share the same price of this offer
 	RegionList  []string
 	Description string
-	Unit        string
-	USD         float64
+	HourlyUSD   float64
```

Comment:
Curious the currency is always in USD?

Reply:
Yes, USD is always the case.

Reply:
I think the question was whether all Cloud providers and all regions use USD.

Reply:
For AWS, the answer is yes. (may be a coincident)